packages/libcom_err.rb

```diff
@@ -11,6 +11,7 @@ class Libcom_err < Package
   depends_on 'e2fsprogs'
 
   def self.install
+    ENV['CREW_SHRINK_ARCHIVE'] = '0'
```

Comment:
Any reason for this here? (and why not put this in arch specific CASE if it is i686 specific?)

Reply:
Yes, because symlink deletes what I want this package to contain. :)